Dwolla.Client.Tests/DwollaClientShould.cs

```diff
@@ -19,7 +19,6 @@ public class DwollaClientShould
     {
         private const string JsonV1 = "application/vnd.dwolla.v1.hal+json";
         private const string RequestId = "some-id";
-        private const string UserAgent = "dwolla-v2-csharp/5.3.0";
```

Comment:
Hey @IsaiahDahlberg, @natehitze could you elaborate on the reason for removing the UserAgent and its test?

Reply:
Added back. I can't recall why I removed it in the first place. I believe we were coming back and forth on something else while bumping the httpClient and this slipped through the cracks